etl/grapher_import.py

```diff
@@ -198,66 +216,54 @@ def upsert_table(table: catalog.Table, dataset_upsert_result: DatasetUpsertResul
             # Not exists, upsert it
             # NOTE: this could be quite inefficient as we upsert source for every variable
             #   optimize this if this turns out to be a bottleneck
-            source_id = _upsert_source_to_db(db, source, dataset_upsert_result.dataset_id)
+            source_id = _upsert_source_to_db(session, source, dataset_upsert_result.dataset_id)
 
-        db_variable_id = db.upsert_variable(
-            short_name=column_name,
-            title=table[column_name].title,
-            source_id=source_id,
-            dataset_id=dataset_upsert_result.dataset_id,
+        variable = gm.Variable(
+            shortName=column_name,
+            name=table[column_name].title,
+            sourceId=source_id,
+            datasetId=dataset_upsert_result.dataset_id,
             description=table[column_name].metadata.description,
-            code=None,
             unit=table[column_name].metadata.unit,
-            short_unit=table[column_name].metadata.short_unit,
+            shortUnit=table[column_name].metadata.short_unit,
             timespan=timespan,
             coverage="",
             display=table[column_name].metadata.display,
-            original_metadata=None,
         )
```

Comment:
Suggest moving the logic to the grapher model:

```python
variable = gm.Variable.from_catalog_table(table, column, timespan)
```

Reply:
Moved it there (in a slightly different way).